community-operators/cluster-logging/clusterlogging.v0.0.1.clusterserviceversion.yaml

```diff
@@ -158,44 +156,6 @@ spec:
           - privileged
           verbs:
           - use
```

Comment:
According to the catalog-operator this is an invalid verb:
```
spec.install.spec.permissions.rules.verbs in body should be one of [*
        assign get list watch create update patch delete deletecollection
        initialize]
```

Reply:
Validation Error in OLM Fixed by: https://github.com/operator-framework/operator-lifecycle-manager/pull/717